LeanplumTests/LeanplumTests.m

```diff
@@ -76,7 +116,7 @@ - (void) testUserAttributesError
         [expectation fulfill];
```

Comment:
we started adding extra assertequal tests to check that the returned errors are as expected, let's add them here too? You can probably see other tests for examples

Reply:
Great point